### PR TITLE
Simplify `evmChainService.latestBlockNum()` and increase test timeout

### PR DESCRIPTION
**Pull request by geoknee**
This got the test passing on my machine.

### CODE REVIEW

File: .github/workflows/go.yml [hunk 1]
```diff
@@ -32,7 +31,7 @@ jobs:
         run: go build -v ./...
 
       - name: Test
-        timeout-minutes: 1
+        timeout-minutes: 30
```
Comment (geoknee) [inline .github/workflows/go.yml:34]: We probably want to cherry-pick this commit to `main`.

File: .github/workflows/testground.yml [hunk 2]
```diff
@@ -2,78 +2,83 @@ name: Run Testground
...
       - name: Update Test Dependency
         run: go get github.com/statechannels/go-nitro@${{github.event.pull_request.head.sha}}
         working-directory: "code/go-nitro-testground"
 
-      - name: Import Test 
-        run:    testground plan import --from ./go-nitro-testground
+      # Bring in any transitive dependencies
+      - name: Run go mod tidy
+        run: go mod tidy
+        working-directory: "code/go-nitro-testground"
```
Comment (geoknee) [inline .github/workflows/testground.yml:33]: We will want to cherry-pick this onto main, too.

**Comment by geoknee** [comment]
Testground run currently failing with many panics like this:

```
panic: 0xd1CE2BaF3756C16B23853b2Bc6626E60313e20f1, error in run loop: objective {Status:1 C:0xc00063f480 finalTurnNum:42 transactionSubmitted:true} cannot handle event {commonEvent:{channelID:[241 67 218 41 241 254 126 128 232 27 155 118 110 145 182 1 11 187 13 187 67 74 28 233 196 211 132 81 65 225 114 5] BlockNum:2806} assetAndAmount:{AssetAddress:0x0000000000000000000000000000000000000000 AssetAmount:+0} NowHeld:+0}`
```

**Comment by kerzhner** [comment]
`TestFevmDirectFund` unit test passes on my machine and takes around 1 minute. Great work @geoknee!

**Pull request merged**